aws-serverless-java-container-springboot3/src/main/java/com/amazonaws/serverless/proxy/spring/SpringDelegatingLambdaContainerHandler.java

```diff
@@ -53,6 +53,10 @@ public SpringDelegatingLambdaContainerHandler() {
     public SpringDelegatingLambdaContainerHandler(Class<?>... startupClasses) {
         this.startupClasses = startupClasses;
         this.mvc = ServerlessMVC.INSTANCE(this.startupClasses);
+        if (System.getenv().containsKey("AWS_LAMBDA_INITIALIZATION_TYPE") 
+        		&& System.getenv().get("AWS_LAMBDA_INITIALIZATION_TYPE").equals("snap-start")) {
```

Comment:
shouldn't that be !"on-demand" to cover "provisioned-concurrency" as well?

Reply:
Perhaps, although I am not quite sure what 'on-demand' is

Reply:
Also, you can change s-c-function version from snapshot to 4.1.1 as we just had a release

Reply:
ahha, got it. will add